algosdk/v2client/algod.py

```diff
@@ -113,6 +113,8 @@ def algod_request(
             try:
                 return json.load(resp)
             except Exception as e:
+                if resp.status == 200:
```

Comment:
We have 5 endpoints that return no response upon success.

[204](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204) might be better, but would entail a breaking change

Reply:
Is this solvable but providing `response_format` = some new field and a new `elif` code-path?